admin-console/src/main/webapp/src/pages/EmailTemplatePage.tsx

```diff
@@ -119,15 +123,14 @@ export const EmailTemplatePage: React.FunctionComponent = () => {
                             type='text'
                             id='description'
                             name="description"
-                            value={ templates?.description }
+                            value={ template?.description }
                             onChange={ handleChange }
                         /></FormGroup>
                     <FormGroup>
                         <Title headingLevel="h2">Data</Title>
                         <CodeEditor
                             isLineNumbersVisible
-                            code={ defaultContentTemplate }
-                            value={ templates?.data}
+                            code={ template?.data }
```

Comment:
Removed the `value` Looks like the CodeEditor does not need it.
Also changed the `code` to use the `template?.data` and initialized it with `defaultContentTemplate` (when creating the state variable)